.develop.sh

```diff
@@ -15,14 +15,27 @@ CONFIGVERSION="$(cat src/config/index.ts | sed "s/^  version:.*/version: '${VERS
 
 # Update the package metadata with the specified version
 PACKAGEJSON="$(jq ".version = \"${VERSION}-beta\"" package.json)" && echo "${PACKAGEJSON}" > package.json
+PACKAGEJSON="$(jq ".version = \"${VERSION}-beta\"" dashboard/package.json)" && echo "${PACKAGEJSON}" > dashboard/package.json
+PACKAGEJSON="$(jq ".version = \"${VERSION}-beta\"" site/package.json)" && echo "${PACKAGEJSON}" > site/package.json
 
 # Make sure the lockfiles are updated, too
 yarn
 
 # Make sure it's all formatted the way we want it
 yarn format
 
+# Same for the dashboard
+cd dashboard
+yarn
+yarn format
```

Comment:
nit: do we need this?

Reply:
Hmm... Does our formatter also format the `package.json` file? I know we've configured formatting to touch `.sql` files, for instance. I put that in there just to be safe that we don't accidentally auto-commit files that break our tests.

Reply:
the `yarn format` inside the `dashboard` dir calls `npx prettier --config .prettierrc 'src/**/*.{ts,tsx}' --write` so should not be a problem

Reply:
Then I can remove it if you want. We'll just need to be careful to re-add it if we ever auto-format the `package.json` files.